app/src/main/java/com/github/multimatum_team/multimatum/CalendarActivity.kt

```diff
@@ -1,54 +1,114 @@
 package com.github.multimatum_team.multimatum
 
+import android.content.Context
+import android.graphics.Rect
 import android.os.Bundle
+import android.view.KeyEvent
+import android.view.MotionEvent
 import android.view.View
+import android.view.inputmethod.InputMethodManager
 import android.widget.CalendarView
+import android.widget.EditText
 import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
 import com.github.multimatum_team.multimatum.model.Deadline
 import com.github.multimatum_team.multimatum.model.DeadlineState
-import com.github.multimatum_team.multimatum.repository.DeadlineRepository
 import com.github.multimatum_team.multimatum.viewmodel.DeadlineListViewModel
 import com.google.android.material.textfield.TextInputEditText
 import dagger.hilt.android.AndroidEntryPoint
-import kotlinx.coroutines.runBlocking
 import java.time.Instant
+import java.time.LocalDate
 import java.time.ZoneId
-import javax.inject.Inject
+
+
+//import javax.inject.Inject
```

Comment:
If this is not useful, you can delete this comment or make it closer to the other import I think.